testing/conftest.py

```diff
@@ -34,6 +34,14 @@ def webtext(base_url, selenium):
             return selenium.find_element_by_tag_name('h1').text
         """)
 
+    testdir.makefile('.cfg', setup="""
+        [tool:pytest]
+        filterwarnings =
+            error::DeprecationWarning
+            ignore:--firefox-\w+ has been deprecated:DeprecationWarning
+            ignore:MarkInfo:DeprecationWarning:pytest_selenium.drivers.firefox:88
+    """)
+
```

Comment:
Nice! What do we need to do to enable failure on warnings?

Reply:
Do you want it to fail on all warnings, not just DeprecationWarnings?

Reply:
Let's discuss in a new issue/pull request. I'm thinking just deprecation warnings, but also interested to know what other warnings we have.